StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1615ElementReturnValueMustBeDocumented.cs

```diff
@@ -18,6 +18,7 @@
     /// <c>&lt;returns&gt;</c> tag.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: I believe this one can be generated, at least as an element containing no text. The result would then produce warning SA1616, but at least the user wouldn't have to type the XML portion manually.

```csharp
/// <returns></returns>
```